scripts/lib/mkcloud-onhost.sh

```diff
@@ -121,7 +121,7 @@ function onhost_cacheclouddata
         for a in $architectures; do
             local suffix
             # Mirror SLES/HA/SOC update + pool repos
-            for suffix in Pool Updates; do
+            for suffix in Pool Updates Updates-test; do
```

Comment:
@guangyee the ha test updates should only be needed if both hacloud is set as well as want_test_updates. TESTHEAD=1 has a unrelated different semantics that for historical reasons also enables want_test_updates.

Is there a problem with my suggestion?